File: Iniciante/Pagina 03/1101/1101.c
#include <stdio.h>
 
int main() {
 
    int a, b, inicio, fim, soma = 0;

    scanf("%d %d", &a, &b);
    while (a > 0 && b > 0) {
        if (a > b) {
            fim = a;
            inicio = b;
        } else {
            fim = b;
            inicio = a;
        }

        for (;inicio <= fim; inicio++) {
            soma += inicio;
            printf("%d ", inicio);
        }
        printf("Sum=%d\n", soma);
        soma = 0;

        scanf("%d %d", &a, &b);
    }
 
    return 0;
}

File: Iniciante/Pagina 03/1101/1101.py
soma = 0
a, b = map(int, input().split())

while a > 0 and b > 0:
    if a > b:
        fim = a
        inicio = b
    else:
        fim = b
        inicio = a
    
    for inicio in range(inicio, fim + 1):
        print("%d " %inicio, end='')
        soma += inicio

    print("Sum=%d" %soma)
    soma = 0

    a, b = map(int, input().split())
